src/App.tsx

```diff
@@ -38,9 +39,17 @@ const App: React.FC = () => {
             <AppInnerContainer>
               <Switch>
                 <Route path="/" exact component={Home} />
-                <Route path="/block-template" exact component={BlockTemplate} />
-                <Route path="/grid-template" exact component={GridTemplate} />
                 <Route path="/login" exact component={Login} />
+                <Route
+                  path="/forgot-password"
```

Comment:
I added an enum `Routes` in a recent update, so you can add this string to that then reference the enum wherever the string is used.